trunk/cli/README.md

```diff
@@ -1,51 +1,82 @@
+- `trunk test` - facilitate the automated unit and integration testing Postgres extensions.
+- `trunk build` - compiles extensions and supports nested dependencies.
+- `trunk publish` - publishes an extension to the registry, making it available for discovery and installation.
+- `trunk install` - download and install the extension distribution, in whichever environment trunk is run.
 
-`trunk test`
-- facilitate the automated unit and integration testing Postgres extensions.
+### 1. `trunk init`
 
-`trunk build`
-- compiles extensions.
-- supports nested dependencies, e.g. installing `extension_a` will automatically install `extension_b` if required.
+### 2. `trunk test`
 
-`trunk publish`
-- publishes an extension to the registry, making it available for discovery and installation.
+### 3. `trunk build`
+e.g. installing extension_a will automatically install extension_b if required
 
-`trunk install`
-- download and install the extension distribution, in whichever environment trunk is run.
+Usage: trunk build [OPTIONS]
 
-### The Registry
+Options:
+- -p, --path <PATH>                [default: .]
+- -o, --output-path <OUTPUT_PATH>  [default: ./.trunk]
+- -h, --help                       Print help
 
-To complement the CLI, we are building a public registry for distributing extension source code and compiled binaries matched to operating system, architecture, Postgres version, and extension version.
+### 4. `trunk publish`
 
-This purpose-built registry would provide a centralized location for developers to share extensions and for users to discover them.
+Usage: trunk publish [OPTIONS] --version <VERSION> <NAME>
 
-### The Website
+Arguments:
+  <NAME>
 
-We will launch a [website](https://pgtrunk.io) to help developers both discover and learn about extensions. This website is key to our success, as it will drive attention, traffic, etc, and lead users to the CLI tool.
+Options:
+-  -v, --version <VERSION>
+-  -f, --file <FILE>
+-  -d, --description <DESCRIPTION>
+-  -D, --documentation <DOCUMENTATION>
+-  -H, --homepage <HOMEPAGE>
+-  -l, --license <LICENSE>
+-  -r, --registry <REGISTRY>            [default: https://registry.pgtrunk.io]
+-  -R, --repository <REPOSITORY>
+-  -h, --help                           Print help
 
-Features will include:
 
-- Extension search and browsing
-- Usage and release metrics, to provide insight into popular and well-maintained extensions
-- User comments and social media streams
-- Benchmarks and tests
-- Version tracking and new release email notifications
+### 5. `trunk install`
+
+Usage: trunk install [OPTIONS] --version <VERSION> <NAME>
+
+Arguments:
+  <NAME>
+
+Options:
+-  -p, --pg-config <PG_CONFIG>
+-  -f, --file <FILE>
+-  -v, --version <VERSION>
+-  -r, --registry <REGISTRY>    [default: https://registry.pgtrunk.io]
+-  -h, --help                   Print help
+
+## Use Case Example
+
+
+
+# Community Involvement
+
+
+
+# Further Reading
```

Comment:
Love these sections. `Community Involvement` might eventually be a `CONTRIBUTING.md` file.